tecken/storage.py

```diff
@@ -113,19 +113,6 @@ def __repr__(self):
             f"backend={self.backend!r}>"
         )
 
-    @property
-    def client(self):
-        """Return a backend-specific client, cached on first access.
-
-        TODO(jwhitlock): Build up StorageBucket API so users don't work directly with
-        the backend-specific clients (bug 1564452).
-        """
-        if not getattr(self, "_client", None):
-            self._client = get_storage_client(
-                endpoint_url=self.endpoint_url, region_name=self.region
-            )
-        return self._client
```

Comment:
This was last used in the code that supported private symbols buckets that weren't accessible via HTTP which I removed in July.